.github/workflows/pr.yml

```diff
@@ -46,5 +46,5 @@ jobs:
           fetch-depth: 0
       - name: pr test
         env:
-          E2E_TESTING_LEVEL: "pr-e2e"
-        run: make pr-test
+          E2E_TESTING_LEVEL: "api"
+        run: make e2e-test
```

Comment:
is this change right? why action `pr-e2e` is changed to `api`

Reply:
This is just for debugging